concepts/switch/about.md

```diff
@@ -0,0 +1,60 @@
+# About
+
+Like other languages, C++ also provides a `switch` statement.
+Switch statements are a shorter way to write long `if ... else if` statements.
+To make a switch, we start by using the keyword `switch` followed by an integer.
+We then declare each one of the conditions with the `case` keyword.
+We can also declare a `default` case, that will run when none of the previous `case` conditions matched.
+Each case should end with a `break` (or a `return`) statement.
```

Comment:
Because this is the `about` doc (_so therefore has more detail/usecases/explaination than the intro_), I'd suggest adding a little more detail around which languages implement `case/switch`, as well as _how_ they implement it.  Not a lengthy thing .. but maybe mentioning that some languages don't have fall-through (Python), and some `match` on more that numeric values (Java, Python, Ruby, PHP), and some implement pattern matching (many functional languages) -- which is .. a bit different?

I suggest this because those coming from other programming languages may or may not be surprised by the C++ behavior.

Reply:
Additionally, maybe some words about why you'd use this form instead of `if-else` (_or when you'd use each??_).

Reply:
That's a good addition, many people will get beef with the compiler and don't get why, because the switch statement in C++17 is rather limited.